src/app/organization/organization-routing.module.ts

```diff
@@ -197,6 +197,10 @@ const routes: Routes = [
           resolve: {
             workingDays: WorkingDaysResolver
           }
+        },
+        {
+          path: 'bulk-import',
+          loadChildren: '../bulk-import/bulk-import.module#BulkImportModule'
```

Comment:
Really appreciate your willingness to learn!!

Reply:
@abhaychawla Thank you :)